src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/ExecutableStatementCountCheck.java

```diff
@@ -95,7 +95,6 @@ public void setMax(int max) {
     @Override
     public void beginTree(DetailAST rootAST) {
         context = new Context(null);
-        contextStack.clear();
```

Comment:
I have tried to launch it on several files, but nothing changed. Maybe they don't suit some condition...